requirements-dev.txt

```diff
@@ -14,6 +14,7 @@ coverage>=6.3
 pylatexenc
 mthree
 scikit-learn
+ddt>=1.2.0,!=1.4.0,!=1.4.3
```

Comment:
nitpicking a bit here but I don't think we need an extra library to test this when a for loop would do the same thing

Reply:
I was making the same standard as Terra, but if you say so, I could write it with for-loop.

Reply:
ah ok, never mind then, everything looks good to me